build/tools/esp32/esptool_py/Makefile

```diff
@@ -74,19 +74,32 @@ $(APP_BIN_UNSIGNED): $(APP_ELF) $(ESPTOOLPY_SRC)
 
 ESPTOOL_ALL_FLASH_ARGS += $(ESPTOOL_BOOTLOADER_FLASH_ARGS)
 
-ALL: flash romfs
+ESPTOOLPY_SUBMODULE := esptool_py/esptool
```

Comment:
Seemingly, this ESPTOOLPY_SUBMODULE is unnecessary, and it's not the exact submodule path of esptool.

Reply:
Fixed, thanks.